CRITICAL - 2024-11-15 09:22:02 --> CodeIgniter\Filters\Exceptions\FilterException: "noauth" filter must have a matching alias defined.
[Method: GET, Route: /]
in SYSTEMPATH\Filters\Filters.php on line 495.
 1 SYSTEMPATH\Filters\Filters.php(495): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('noauth')
 2 SYSTEMPATH\Filters\Filters.php(547): CodeIgniter\Filters\Filters->enableFilter('noauth', 'before')
 3 SYSTEMPATH\CodeIgniter.php(469): CodeIgniter\Filters\Filters->enableFilters([...], 'before')
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 09:25:14 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:25:16 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:26:01 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:26:58 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:26:59 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:27:24 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:27:25 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-11-15 09:28:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\projects\Chat\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'chatdemo', 3306, '', 0)
#1 C:\xampp\htdocs\projects\Chat\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\projects\Chat\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projects\Chat\system\Database\BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\projects\Chat\system\Database\BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\projects\Chat\system\Database\MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\projects\Chat\system\Database\MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\projects\Chat\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\projects\Chat\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\projects\Chat\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\projects\Chat\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\projects\Chat\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\projects\Chat\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\projects\Chat\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\projects\Chat\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projects\Chat\system\Database\BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\projects\Chat\system\Database\BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\projects\Chat\system\Database\MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\projects\Chat\system\Database\MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\projects\Chat\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\projects\Chat\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\projects\Chat\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\projects\Chat\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\projects\Chat\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\projects\Chat\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-11-15 09:29:23 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:29:28 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:33:58 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:34:30 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:34:31 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:34:33 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:35:29 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:35:30 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:53:55 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:54:25 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users/register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:54:40 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: 1/register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:54:42 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: 1/]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:54:55 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:55:26 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:55:31 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:55:53 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:55:54 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:55:57 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:56:00 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: login]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:57:19 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:57:46 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: chat]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:57:59 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: logout]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:58:21 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users/register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:58:49 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users/register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:58:50 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users/register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:58:50 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users/register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:58:51 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users/register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:58:51 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users/register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:58:51 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users/register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:58:51 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users/register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:59:42 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users/register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:59:55 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users/register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 09:59:56 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users/register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:01:52 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:02:26 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:02:34 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:02:37 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:02:52 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:03:32 --> ErrorException: Declaration of App\Filters\UsersCheck::after(CodeIgniter\HTTP\RequestInterface $request, CodeIgniter\HTTP\ResponseInterface $response) must be compatible with CodeIgniter\Filters\FilterInterface::after(CodeIgniter\HTTP\RequestInterface $request, CodeIgniter\HTTP\ResponseInterface $response, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 30.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:03:33 --> ErrorException: Declaration of App\Filters\UsersCheck::after(CodeIgniter\HTTP\RequestInterface $request, CodeIgniter\HTTP\ResponseInterface $response) must be compatible with CodeIgniter\Filters\FilterInterface::after(CodeIgniter\HTTP\RequestInterface $request, CodeIgniter\HTTP\ResponseInterface $response, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 30.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:05:28 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:08:18 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: /]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:08:22 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: login]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:08:26 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: register]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:08:29 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:09:00 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:09:01 --> ErrorException: Declaration of App\Filters\UsersCheck::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: users]
in APPPATH\Filters\UsersCheck.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:09:38 --> ErrorException: Declaration of App\Filters\Noauth::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: register]
in APPPATH\Filters\Noauth.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:29:28 --> ErrorException: Declaration of App\Filters\Auth::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: dashboard]
in APPPATH\Filters\Auth.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:29:39 --> ErrorException: Declaration of App\Filters\Auth::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: chat]
in APPPATH\Filters\Auth.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:29:55 --> ErrorException: Declaration of App\Filters\Auth::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: profile]
in APPPATH\Filters\Auth.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:30:18 --> ErrorException: Declaration of App\Filters\Auth::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: profile]
in APPPATH\Filters\Auth.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:30:19 --> ErrorException: Declaration of App\Filters\Auth::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: profile]
in APPPATH\Filters\Auth.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:30:54 --> ErrorException: Declaration of App\Filters\Auth::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: profile]
in APPPATH\Filters\Auth.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:30:55 --> ErrorException: Declaration of App\Filters\Auth::before(CodeIgniter\HTTP\RequestInterface $request) must be compatible with CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null)
[Method: GET, Route: profile]
in APPPATH\Filters\Auth.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:34:52 --> CodeIgniter\Validation\Exceptions\ValidationException: "validateUser" is not a valid rule.
[Method: POST, Route: /]
in SYSTEMPATH\Validation\Validation.php on line 348.
 1 SYSTEMPATH\Validation\Validation.php(348): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('validateUser')
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', [...], [...], 'password')
 3 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 4 APPPATH\Controllers\Users.php(28): CodeIgniter\Controller->validate([...], [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 10:34:52 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Validation\Exceptions\ValidationException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Validation\Exceptions\ValidationException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Validation\Exceptions\ValidationException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Validation\Exceptions\ValidationException
"validateUser" is not a valid rule.
#0 C:\xampp\htdocs\projects\Chat\system\Validation\Validation.php(348): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('validateUser')
#1 C:\xampp\htdocs\projects\Chat\system\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', Array, Array, 'password')
#2 C:\xampp\htdocs\projects\Chat\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(28): CodeIgniter\Controller->validate(Array, Array)
#4 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#5 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#6 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#11 {main}
[Method: POST, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:40:13 --> CodeIgniter\Validation\Exceptions\ValidationException: "validateUser" is not a valid rule.
[Method: POST, Route: /]
in SYSTEMPATH\Validation\Validation.php on line 348.
 1 SYSTEMPATH\Validation\Validation.php(348): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('validateUser')
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', [...], [...], 'password')
 3 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 4 APPPATH\Controllers\Users.php(24): CodeIgniter\Controller->validate([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 10:40:13 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Validation\Exceptions\ValidationException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Validation\Exceptions\ValidationException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Validation\Exceptions\ValidationException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Validation\Exceptions\ValidationException
"validateUser" is not a valid rule.
#0 C:\xampp\htdocs\projects\Chat\system\Validation\Validation.php(348): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('validateUser')
#1 C:\xampp\htdocs\projects\Chat\system\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', Array, Array, 'password')
#2 C:\xampp\htdocs\projects\Chat\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(24): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#5 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#6 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#11 {main}
[Method: POST, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:40:15 --> CodeIgniter\Validation\Exceptions\ValidationException: "validateUser" is not a valid rule.
[Method: POST, Route: /]
in SYSTEMPATH\Validation\Validation.php on line 348.
 1 SYSTEMPATH\Validation\Validation.php(348): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('validateUser')
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', [...], [...], 'password')
 3 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 4 APPPATH\Controllers\Users.php(24): CodeIgniter\Controller->validate([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 10:40:15 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Validation\Exceptions\ValidationException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Validation\Exceptions\ValidationException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Validation\Exceptions\ValidationException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Validation\Exceptions\ValidationException
"validateUser" is not a valid rule.
#0 C:\xampp\htdocs\projects\Chat\system\Validation\Validation.php(348): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('validateUser')
#1 C:\xampp\htdocs\projects\Chat\system\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', Array, Array, 'password')
#2 C:\xampp\htdocs\projects\Chat\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(24): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#5 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#6 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#11 {main}
[Method: POST, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:40:30 --> CodeIgniter\Validation\Exceptions\ValidationException: "validateUser" is not a valid rule.
[Method: POST, Route: /]
in SYSTEMPATH\Validation\Validation.php on line 348.
 1 SYSTEMPATH\Validation\Validation.php(348): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('validateUser')
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', [...], [...], 'password')
 3 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 4 APPPATH\Controllers\Users.php(24): CodeIgniter\Controller->validate([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 10:40:30 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Validation\Exceptions\ValidationException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Validation\Exceptions\ValidationException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Validation\Exceptions\ValidationException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Validation\Exceptions\ValidationException
"validateUser" is not a valid rule.
#0 C:\xampp\htdocs\projects\Chat\system\Validation\Validation.php(348): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('validateUser')
#1 C:\xampp\htdocs\projects\Chat\system\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', Array, Array, 'password')
#2 C:\xampp\htdocs\projects\Chat\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(24): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#5 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#6 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#11 {main}
[Method: POST, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:41:11 --> CodeIgniter\Validation\Exceptions\ValidationException: "validateUser" is not a valid rule.
[Method: POST, Route: /]
in SYSTEMPATH\Validation\Validation.php on line 348.
 1 SYSTEMPATH\Validation\Validation.php(348): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('validateUser')
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', [...], [...], 'password')
 3 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 4 APPPATH\Controllers\Users.php(24): CodeIgniter\Controller->validate([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 10:41:11 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Validation\Exceptions\ValidationException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Validation\Exceptions\ValidationException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Validation\Exceptions\ValidationException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Validation\Exceptions\ValidationException
"validateUser" is not a valid rule.
#0 C:\xampp\htdocs\projects\Chat\system\Validation\Validation.php(348): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('validateUser')
#1 C:\xampp\htdocs\projects\Chat\system\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', Array, Array, 'password')
#2 C:\xampp\htdocs\projects\Chat\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(24): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#5 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#6 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#11 {main}
[Method: POST, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:41:12 --> CodeIgniter\Validation\Exceptions\ValidationException: "validateUser" is not a valid rule.
[Method: POST, Route: /]
in SYSTEMPATH\Validation\Validation.php on line 348.
 1 SYSTEMPATH\Validation\Validation.php(348): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('validateUser')
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', [...], [...], 'password')
 3 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 4 APPPATH\Controllers\Users.php(24): CodeIgniter\Controller->validate([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 10:41:12 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Validation\Exceptions\ValidationException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Validation\Exceptions\ValidationException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Validation\Exceptions\ValidationException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Validation\Exceptions\ValidationException
"validateUser" is not a valid rule.
#0 C:\xampp\htdocs\projects\Chat\system\Validation\Validation.php(348): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('validateUser')
#1 C:\xampp\htdocs\projects\Chat\system\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', Array, Array, 'password')
#2 C:\xampp\htdocs\projects\Chat\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(24): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#5 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#6 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#11 {main}
[Method: POST, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:42:06 --> ErrorException: Undefined variable $session
[Method: POST, Route: /]
in APPPATH\Controllers\Users.php on line 33.
 1 APPPATH\Controllers\Users.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $session', 'C:\\xampp\\htdocs\\projects\\Chat\\app\\Controllers\\Users.php', 33)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 10:42:06 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined variable $session
#0 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 33)
#1 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#2 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#6 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#7 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#8 {main}
[Method: POST, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:48:07 --> ErrorException: Undefined variable $password
[Method: POST, Route: /]
in APPPATH\Controllers\Users.php on line 32.
 1 APPPATH\Controllers\Users.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $password', 'C:\\xampp\\htdocs\\projects\\Chat\\app\\Controllers\\Users.php', 32)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 10:48:07 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined variable $password
#0 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 32)
#1 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#2 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#6 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#7 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#8 {main}
[Method: POST, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:49:22 --> ErrorException: Trying to access array offset on value of type null
[Method: POST, Route: /]
in APPPATH\Controllers\Users.php on line 32.
 1 APPPATH\Controllers\Users.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\projects\\Chat\\app\\Controllers\\Users.php', 32)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 10:49:22 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Trying to access array offset on value of type null
#0 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp\\htdocs...', 32)
#1 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#2 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#6 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#7 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#8 {main}
[Method: POST, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:56:23 --> ErrorException: Undefined variable $token
[Method: POST, Route: /]
in APPPATH\Controllers\Users.php on line 40.
 1 APPPATH\Controllers\Users.php(40): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $token', 'C:\\xampp\\htdocs\\projects\\Chat\\app\\Controllers\\Users.php', 40)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 10:56:23 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined variable $token
#0 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(40): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 40)
#1 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#2 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#6 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#7 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#8 {main}
[Method: POST, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:57:02 --> ErrorException: Undefined array key "firstname"
[Method: GET, Route: profile]
in APPPATH\Views\profile.php on line 5.
 1 APPPATH\Views\profile.php(5): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "firstname"', 'C:\\xampp\\htdocs\\projects\\Chat\\app\\Views\\profile.php', 5)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\projects\\Chat\\app\\Views\\profile.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('profile', [], true)
 5 APPPATH\Controllers\Users.php(160): view('profile')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->profile()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 10:57:02 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined array key "firstname"
#0 C:\xampp\htdocs\projects\Chat\app\Views\profile.php(5): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 5)
#1 C:\xampp\htdocs\projects\Chat\system\View\View.php(234): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\projects\Chat\system\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\projects\Chat\system\Common.php(1171): CodeIgniter\View\View->render('profile', Array, true)
#4 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(160): view('profile')
#5 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->profile()
#6 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#7 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#12 {main}
[Method: GET, Route: profile]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:58:29 --> ErrorException: Trying to access array offset on value of type null
[Method: POST, Route: /]
in APPPATH\Controllers\Users.php on line 29.
 1 APPPATH\Controllers\Users.php(29): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\projects\\Chat\\app\\Controllers\\Users.php', 29)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 10:58:29 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Trying to access array offset on value of type null
#0 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(29): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp\\htdocs...', 29)
#1 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#2 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#6 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#7 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#8 {main}
[Method: POST, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 10:58:54 --> ErrorException: Trying to access array offset on value of type null
[Method: POST, Route: /]
in APPPATH\Controllers\Users.php on line 32.
 1 APPPATH\Controllers\Users.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\projects\\Chat\\app\\Controllers\\Users.php', 32)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 10:58:54 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Trying to access array offset on value of type null
#0 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp\\htdocs...', 32)
#1 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#2 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#6 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#7 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#8 {main}
[Method: POST, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 11:00:30 --> ErrorException: Undefined variable $session
[Method: POST, Route: /]
in APPPATH\Controllers\Users.php on line 31.
 1 APPPATH\Controllers\Users.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $session', 'C:\\xampp\\htdocs\\projects\\Chat\\app\\Controllers\\Users.php', 31)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 11:00:30 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined variable $session
#0 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 31)
#1 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#2 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#6 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#7 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#8 {main}
[Method: POST, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 11:01:18 --> ErrorException: Undefined array key "firstname"
[Method: GET, Route: profile]
in APPPATH\Views\profile.php on line 5.
 1 APPPATH\Views\profile.php(5): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "firstname"', 'C:\\xampp\\htdocs\\projects\\Chat\\app\\Views\\profile.php', 5)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\projects\\Chat\\app\\Views\\profile.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('profile', [], true)
 5 APPPATH\Controllers\Users.php(166): view('profile')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->profile()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 11:01:18 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined array key "firstname"
#0 C:\xampp\htdocs\projects\Chat\app\Views\profile.php(5): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 5)
#1 C:\xampp\htdocs\projects\Chat\system\View\View.php(234): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\projects\Chat\system\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\projects\Chat\system\Common.php(1171): CodeIgniter\View\View->render('profile', Array, true)
#4 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(166): view('profile')
#5 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->profile()
#6 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#7 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#12 {main}
[Method: GET, Route: profile]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 11:06:35 --> ErrorException: Undefined array key "token"
[Method: GET, Route: /]
in APPPATH\Controllers\Users.php on line 14.
 1 APPPATH\Controllers\Users.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "token"', 'C:\\xampp\\htdocs\\projects\\Chat\\app\\Controllers\\Users.php', 14)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 11:06:35 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined array key "token"
#0 C:\xampp\htdocs\projects\Chat\app\Controllers\Users.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 14)
#1 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Users->index()
#2 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#6 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#7 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#8 {main}
[Method: GET, Route: /]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 12:10:55 --> ErrorException: Undefined variable $userModel
[Method: GET, Route: chat]
in APPPATH\Controllers\Chat.php on line 14.
 1 APPPATH\Controllers\Chat.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $userModel', 'C:\\xampp\\htdocs\\projects\\Chat\\app\\Controllers\\Chat.php', 14)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Chat->index()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Chat))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 12:10:55 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined variable $userModel
#0 C:\xampp\htdocs\projects\Chat\app\Controllers\Chat.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 14)
#1 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Chat->index()
#2 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Chat))
#3 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#6 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#7 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#8 {main}
[Method: GET, Route: chat]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-15 12:11:14 --> TypeError: CodeIgniter\Database\BaseBuilder::where(): Argument #3 ($escape) must be of type ?bool, string given, called in C:\xampp\htdocs\projects\Chat\system\Model.php on line 930
[Method: GET, Route: chat]
in SYSTEMPATH\Database\BaseBuilder.php on line 715.
 1 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->where('id', '!=', '1')
 2 APPPATH\Controllers\Chat.php(14): CodeIgniter\Model->__call('where', [...])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Chat->index()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Chat))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\projects\\Chat\\public\\index.php')
CRITICAL - 2024-11-15 12:11:14 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\projects\Chat\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\projects\Chat\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\projects\Chat\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(TypeError), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\projects\Chat\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(TypeError), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(TypeError))
#5 {main}
  thrown
【Previous Exception】
TypeError
CodeIgniter\Database\BaseBuilder::where(): Argument #3 ($escape) must be of type ?bool, string given, called in C:\xampp\htdocs\projects\Chat\system\Model.php on line 930
#0 C:\xampp\htdocs\projects\Chat\system\Model.php(930): CodeIgniter\Database\BaseBuilder->where('id', '!=', '1')
#1 C:\xampp\htdocs\projects\Chat\app\Controllers\Chat.php(14): CodeIgniter\Model->__call('where', Array)
#2 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(933): App\Controllers\Chat->index()
#3 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Chat))
#4 C:\xampp\htdocs\projects\Chat\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\projects\Chat\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\projects\Chat\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#7 C:\xampp\htdocs\projects\Chat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#8 C:\xampp\htdocs\projects\Chat\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#9 {main}
[Method: GET, Route: chat]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
